pset-functions.js

```diff
@@ -91,6 +124,31 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+function wordCounter (phrase) {
```

Comment:
🛠 Read the comment below and update your code accordingly

@Defy1x nice logic, however, your indentation is off - please correct the indentation issues

Reply:
Done